core-plugins/src/test/java/co/cask/hydrator/plugin/transform/PythonEvaluatorTest.java

```diff
@@ -229,27 +229,6 @@ public void testComplex() throws Exception {
       Schema.Field.of("inner1", inner1Schema)
     );
 
-    /*
-    {
-      "complex": {
-        "num": 8,
-        "inner1": {
```

Comment:
Since there was no TODO associated with the comment so it was not clear why the comment was there